### PR TITLE
UML-2005: Make it clear older older LPAs is part of a trial

### PR DESCRIPTION
**Pull request by SeemaMenon**
Fixes UML-2005

## Approach

Content addition with translations

## Learning

_Any tips and tricks, blog posts or tools which helped you. Plus anything notable you've discovered about the Use a Lasting Power of Attorney service_

## Checklist

* [X] I have performed a self-review of my own code
* [X] I have added welsh translation tags and updated translation files
* [ ] I have run an accessibility tool on any pages I have made changes to and fixed any issues found
* [ ] The product team have tested these changes

### CODE REVIEW

**Comment by SeemaMenon** [comment]
Screenshot 2022-01-17 at 16 07 19

**Comment by SeemaMenon** [comment]
Screenshot 2022-01-17 at 16 33 47

**Comment by SeemaMenon** [comment]
Screenshot 2022-01-17 at 16 44 58

File: service-front/app/src/Actor/templates/actor/home-page.html.twig [hunk 1]
```diff
@@ -44,10 +44,18 @@
                     <p class="govuk-body">{% trans %}If you're named on more than one LPA, you can add all those LPAs to your account.{% endtrans %}</p>
 
                     <button data-prevent-double-click="true" type="submit" class="govuk-button">{% trans %}Continue{% endtrans %}</button>
+
+                    {% if feature_enabled('allow_older_lpas') %}
+                    <p class="govuk-body">
+                        <span>&#42;</span>
+                        {% trans %}
+                            Adding LPAs registered between 1 January 2016 and 31 August 2019 is on a trial basis only, and may be paused temporarily.However, once your LPA is added you will always be able to use it in this service.
```
Comment (cooperaj) [inline service-front/app/src/Actor/templates/actor/home-page.html.twig:52]:
```suggestion
                            Adding LPAs registered between 1 January 2016 and 31 August 2019 is on a trial basis only, and may be paused temporarily. However, once your LPA is added you will always be able to use it in this service.
```

File: service-front/app/src/Actor/templates/actor/home-page.html.twig [hunk 2]
```diff
@@ -44,10 +44,18 @@
                     <p class="govuk-body">{% trans %}If you're named on more than one LPA, you can add all those LPAs to your account.{% endtrans %}</p>
 
                     <button data-prevent-double-click="true" type="submit" class="govuk-button">{% trans %}Continue{% endtrans %}</button>
+
+                    {% if feature_enabled('allow_older_lpas') %}
+                    <p class="govuk-body">
+                        <span>&#42;</span>
```
Comment (cooperaj) [inline service-front/app/src/Actor/templates/actor/home-page.html.twig:50]: I think there are proper ways of doing HTML footnotes - as accessibility needs to be taken into account.

**Comment by SeemaMenon** [comment]
Screenshot 2022-01-18 at 11 34 16
Screenshot 2022-01-18 at 11 37 21

**Comment by SeemaMenon** [comment]
Screenshot 2022-01-18 at 11 44 11

File: service-front/app/src/Actor/templates/actor/home-page.html.twig (hunk 2 above)
Comment (SeemaMenon) [reply service-front/app/src/Actor/templates/actor/home-page.html.twig]: Removed after our discussion.

**Comment by SeemaMenon** [comment]
Screenshot 2022-01-18 at 11 37 21

**Review by lukejolliffe — APPROVED** [review]

**Pull request merged**